### PR TITLE
Fix label in collections

### PR DESCRIPTION
**Pull request by nedmsmith**
updated cmw-collection.cddl to include a label.
modified prose in section 3.3.1 to better capture the need for integrity protection of label information.

Addresses issue #104

### CODE REVIEW

**Comment by thomas-fossati** [comment]
@carl-wallace could you please check that the proposed text addresses your concern?

File: draft-ietf-rats-msg-wrap.md [hunk 1]
```diff
@@ -284,8 +284,7 @@ A CMW collection carried in a `"cmw"` CWT claim ({{iana-cwt}}) MUST be a `cbor-c
 ### CMW Collections' role in composite Attester topology
 
 A CMW Collection's tree structure is not required to be a spanning tree of the system's composite Attester topology.
-If a label changes Verifier state beyond a "hint" (e.g., for better Verifier performance or human comprehension), we say that it carries semantic content.
-When a label carries semantic content that is not bound to other forms of evidence contained in the collection, the collection SHOULD be signed by an attestation key, e.g., by including the collection in a signed EAT {{-rats-eat}}.
+If the label includes semantic content for a Verifier (e.g., for better Verifier performance or human comprehension), the collection SHOULD be integrity protected, e.g., by including it in a signed token such as a CWT or JWT.
```
Comment (thomas-fossati) [inline draft-ietf-rats-msg-wrap.md:287]: * use "labels" plural
* split sentence into two

```suggestion
If the labels carry semantic content for a Verifier (e.g. to improve Verifier performance or aid human comprehension), the collection SHOULD be integrity protected. For example, by including it in a signed token such as a CWT or JWT.
```

**Review by thomas-fossati — COMMENTED** [review]
LGTM, I've inlined a small editorial tweak.

File: draft-ietf-rats-msg-wrap.md [hunk 2]
```diff
@@ -284,8 +284,7 @@ A CMW collection carried in a `"cmw"` CWT claim ({{iana-cwt}}) MUST be a `cbor-c
 ### CMW Collections' role in composite Attester topology
 
 A CMW Collection's tree structure is not required to be a spanning tree of the system's composite Attester topology.
-If a label changes Verifier state beyond a "hint" (e.g., for better Verifier performance or human comprehension), we say that it carries semantic content.
-When a label carries semantic content that is not bound to other forms of evidence contained in the collection, the collection SHOULD be signed by an attestation key, e.g., by including the collection in a signed EAT {{-rats-eat}}.
+If the labels carry semantic content for a Verifier (e.g. to improve Verifier performance or aid human comprehension), the collection SHOULD be integrity protected. For example, by including it in a signed token such as a CWT or JWT.
```
Comment (deeglaze) [inline draft-ietf-rats-msg-wrap.md:287]: The "for example" change is a sentence fragment.
```suggestion
If the labels carry semantic content for a Verifier (e.g. to improve Verifier performance or aid human comprehension), the collection SHOULD be integrity protected.
For example, the collection can be integrity protected by including it in a signed token such as a CWT or JWT.
```

**Comment by thomas-fossati** [comment]
@carl-wallace if you have any issues with the proposed text, or have clarifying suggestions, please shout.  Otherwise, we are going to merge it shortly.

**Review by thomas-fossati — APPROVED** [review]

**Review by carl-wallace — APPROVED** [review]

**Pull request merged**